examples/ota-requestor-app/linux/main.cpp

```diff
@@ -25,9 +25,11 @@
 
 #include "LinuxOTAImageProcessor.h"
 #include "LinuxOTARequestorDriver.h"
+#include "app/clusters/ota-requestor/BDXDownloader.h"
 #include "app/clusters/ota-requestor/OTADownloader.h"
```

Comment:
nit: Won't need to include `OTADownloader.h` anymore

Reply:
sure, will remove